gengokit/template/NAME-service/NAME-server/server_main.gotemplate

```diff
@@ -74,7 +74,7 @@ func main() {
 	// Wrap Endpoints with middlewares. See ../middlewares/endpoints.go
 	endpoints := svc.Endpoints{
 	{{range $i := .Service.Methods -}}
-		{{$i.Name}}Endpoint:    middlewares.WrapEndpoint({{ToLower $i.Name}}Endpoint),
+		{{$i.Name}}Endpoint:    {{ToLower $i.Name}}Endpoint,
```

Comment:
`formatCode` is what I was looking briefly for, but did not find. This is fine then. I certainly agree that it improves readability.